narwhals/_arrow/series.py

```diff
@@ -163,6 +163,10 @@ def sum(self) -> int:
         pc = get_pyarrow_compute()
         return pc.sum(self._native_series)  # type: ignore[no-any-return]
 
+    def round(self) -> int:
+        pc = get_pyarrow_compute()
+        return pc.round(self._native_series)  # type: ignore[no-any-return]
```

Comment:
`round` method should take number of digits as well

```suggestion
    def round(self, decimals: int) -> int:
        pc = get_pyarrow_compute()
        return pc.round(self._native_series, n_digits=decimals)  # type: ignore[no-any-return]
```